src/documentation/pages/components/tabs/examples/_contained_component.vue

```diff
@@ -0,0 +1,104 @@
+<template lang="pug">
+  <ComponentExample title="Components contained" id="components-contained-lumen-centurylink" :tabs="exampleTabs">
+    p.-text(slot="example-description")
+      | As navigation component is built from other primitive Chi components, most of the components behavior
+      | can be replicated on the navigation component. For example, you can use the <code>-animate</code> class
+      | on the dropdowns to make the chevron rotate when activated.
+
+    .-p--0(slot="example")
```

Comment:
is this -p--0 padding needed? if so please add it to padding="" in ComponentExample